backstory/src/main/java/com/google/sps/perspective/PerspectiveManager.java

```diff
@@ -14,11 +14,32 @@
 
 package com.google.sps.perspective;
 
+import au.com.origma.perspectiveapi.v1alpha1.models.AttributeType;
+
 /**
  * Handles analysis of stories for content and provides a recommendation on
  * the "best" appropriate story (if multiple).
- */
-interface PerspectiveManager {
+ */ 
+ interface PerspectiveManager {
+  /** an array of all the types we want analysis on */
+  public static final AttributeType[] REQUESTED_ATTRIBUTES = {
+      AttributeType.ATTACK_ON_AUTHOR,
+      AttributeType.ATTACK_ON_COMMENTER, 
+      AttributeType.FLIRTATION, 
+      AttributeType.IDENTITY_ATTACK,
+      AttributeType.INCOHERENT, 
+      AttributeType.INSULT, 
+      AttributeType.LIKELY_TO_REJECT,
+      AttributeType.OBSCENE, 
+      AttributeType.PROFANITY, 
+      AttributeType.SEVERE_TOXICITY,
+      AttributeType.SEXUALLY_EXPLICIT, 
+      AttributeType.SPAM, 
+      AttributeType.THREAT,
+      AttributeType.TOXICITY, 
+      AttributeType.UNSUBSTANTIAL
+    };
```

Comment:
I'm curious why this needed to be in the interface?

Reply:
I needed to place these values somewhere so we could have a centralized list of which attributes we were analyzing for. I thought the interface might be a good place because ```PerspectiveManager.REQUIRED_ATTRIBUTES``` seemed like the most logical place they could be put. However, I'm opening to moving this. @ykjchen Do you have a suggestion? Should I place it in ```PerspectiveManagerImpl``` instead?

Reply:
Do you know yet what objects will need that list? The goal is in general to reduce visibility as much as possible, so that things are exposed only to the classes that need access.

Reply:
After an offline conversation, as PerspectiveManagerImpl is the only one that needs the class, I will move it there.

Reply:
Done